dev-requirements.txt

```diff
@@ -1,13 +1,13 @@
-isort==4.3.21
+isort==5.13.2
 tox
 wheel
 bumpversion
 twine
 black
 flake8
 pylint
+tomli==2.0.1
 cython
 sphinx
 sphinx-rtd-theme
-mypy==0.770
--r requirements.txt
```

Comment:
pyproject.toml does not support recursive package installation

Reply:
How do transitive dependencies get installed then?

Reply:
I mean to say pyproject.toml requires a dependency txt file to list packages directly, and not point to another dependency file.

Transitive dependencies are still handled by pip during installation. The requiremnts.txt file is listed as core dependencies in the pyprojec.toml,  and pip will install the transitive dependencies. The dev-requirements and test-requirements are listed as optional dependencies

Reply:
Thank you for the clarification!